Python/Tests/DjangoTests/DjangoTemplateParserTests.cs

```diff
@@ -26,13 +26,15 @@
 using Microsoft.VisualStudio.Text;
 using Microsoft.PythonTools.Django.TemplateParsing.DjangoBlocks;
 using Microsoft.PythonTools.Django.TemplateParsing;
+using TestUtilities;
+using Classification = Microsoft.PythonTools.Django.TemplateParsing.Classification;
```

Comment:
there doesn't seem to be any other change in this file related to this using, is it unnecessary?

Reply:
Required because adding TestUtilities causes ambiguity for "Classification" which is a imported in "…Django.TemplateParsing" and TestUtilities.

![image](https://user-images.githubusercontent.com/3315992/66006012-07c83500-e462-11e9-889b-e56152238e6f.png)